### PR TITLE
[FrameworkBundle][config] allow multiple fallback locales.

### PR DESCRIPTION
**Pull request by aitboudad**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Fixed tickets | #13373 |
| Doc PR | symfony/symfony-docs#4834 |
| Tests pass? | yes |
| License | MIT |

### CODE REVIEW

**Comment by hacfi** [comment]
:+1:

Not sure if this is considered bug fix or new feature though. We’ll see.

**Comment by fabpot** [comment]
I would consider it as a bug fix.

:+1:

**Comment by WouterJ** [comment]
Fyi, a PR to the docs is submitted

**Comment by stof** [comment]
The XML schema needs to be updated to support multiple fallbacks too

**Comment by aitboudad** [comment]
@stof done

**Comment by fabpot** [comment]
While we are at it, what about allowing `fallbacks` as well like done for many other config items where we can have more than one items?

**Comment by stof** [comment]
We should indeed have ``->arrayNode('fallbacks')`` and a ``fixXmlConfig('fallback')`` on the parent

**Comment by aitboudad** [comment]
done

File: src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/translator_fallback.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                        http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+    <framework:config secret="s3cr3t">
+        <framework:translator enabled="true">
+            <framework:fallback>en</framework:fallback>
+            <framework:fallback>fr</framework:fallback>
+        </framework:translator>
+    </framework:config>
+</container>
```
Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/translator_fallback.xml]: This file does not seem to be used anywhere in the tests. Have you forgot to commit them? Tests should also be done for other formats like YAML.

**Comment by aitboudad** [comment]
@fabpot I added the missing files.

**Comment by fabpot** [comment]
Thank you @aitboudad.

File: src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php [hunk 2]
```diff
@@ -206,6 +206,14 @@ public function testTranslator()
         $this->assertEquals(array('fr'), $calls[0][1][0]);
     }
 
+    public function testTranslatorMultipleFullback()
```
Comment (jdecoster) [inline src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php]: pondering why it's 'Fullback' and not 'Fallback'

Comment (jakzal) [inline src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php]: Would you mind to submit a PR fixing this typo? :)

Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php]: done in 87c0659

**Pull request merged**